src/zc/buildout/download.py

```diff
@@ -24,10 +25,10 @@
     from urllib.parse import urlparse
     from urllib import request
 
-    class PatchedURLopener(FancyURLopener):
-        http_error_default = URLopener.http_error_default
-
-    request._urlopener = PatchedURLopener()  # Ook! Monkey patch!
+    if sys.version_info[:2] < (3, 3):
+        class PatchedURLopener(FancyURLopener):
+            http_error_default = URLopener.http_error_default
+        request._urlopener = PatchedURLopener()  # Ook! Monkey patch!
```

Comment:
I am not sure that this change is needed at all. If `_urlopener` is not used any more in Python's 3.3 `request`, adding it should be innocuous.